Cargo.toml

```diff
@@ -17,6 +17,7 @@ crossterm = { version = "0.26.1", features = ["use-dev-tty"] }
 ratatui = "0.21.0"
 regex = { version = "1.8.2", default-features = false, features = ["std", "perf", "unicode-case" ] }
 arboard = { version = "3.2.0", default-features = false }
+nix = { version = "0.26", default-features = false, features = ["process"] }
```

Comment:
* Let's make this dep [unix-specific](https://github.com/dbus2/zbus/blob/main/zbus/Cargo.toml#L82).
* Please split into separate commit. Otherwise, I'll not remember to mention in release notes.

Reply:
> Please split into separate commit

Which part should I split into a separate commit?

Reply:
This line: addition of `nix` dep.

Reply:
> mention in release notes

I didn't know that addition of a dependency is such a major change. We can just use `libc::daemon` directly btw, just with one safe `unsafe {}`.

Reply:
> I didn't know that addition of a dependency is such a major change.

It's major enough to mention in release notes. :) Maybe not so important for apps though.

> We can just use `libc::daemon` directly btw, just with one safe `unsafe {}`.

Why not use use [the popular crate](https://crates.io/crates/daemonize) for it?

Reply:
> Why not use use [the popular crate](https://crates.io/crates/daemonize) for it?

It has waaay too many functionality for what we are doing here.

Reply:
It will not simplify anything in this case.

Reply:
* It's < 1000 LOC.
* has only libc as a dep. `nix` brings in `bitflags` and `cfg-if` and `static_assertions`.
* does exactly what we need.
* abstracts us from `unsafe`.

Reply:
> nix brings in bitflags and cfg-if and static_assertions.

That's not much of an argument on it's own though since these deps are also coming from other deps.

Reply:
From my understanding, using this crate would address

> /// Note that it is possible to make it work without exiting the process, but it would require an
> /// `unsafe { fork() }`. However, in this programm this is simply not needed.

but I don't think this will ever be needed...